base_gs1_barcode/__manifest__.py

```diff
@@ -2,7 +2,7 @@
 {
     'name': 'GS1 Barcode API',
     'summary': 'Decoding API for GS1-128 (aka UCC/EAN-128) and GS1-Datamatrix',
-    'version': '11.0.1.0.1',
+    'version': '12.0.1.0.1',
```

Comment:
As per https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-12.0, it should be `12.0.1.0.0`

Reply:
Done! Thanks for your review